src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/executor.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Executor
+{
+    public class Executor
+    {
+        public Setup openFile(string filePath)
+        {
+            string jsonFile = string.Empty;
```

Comment:
You can move the declaration of jsonFile into the next scope.  ie, delete this line and add the declaration to "string jsonFile = File.ReadAllText(filePath);  In this way, you're not declaring an unused variable if filePath doesn't exist.